councilmatic/settings_deployment.py.example

```diff
@@ -80,3 +80,6 @@ LOGGING = {
 
 # Set to False in production!
 SHOW_TEST_EVENTS = True
+
+SMART_LOGIC_KEY = 'smartlogic api key'
+SMART_LOGIC_ENVIRONMENT = '0ef5d755-1f43-4a7e-8b06-7591bed8d453'
```

Comment:
based on what you said, i assume this doesn't need to be secret—just double-checking

Reply:
ya, this is just the url. i made it configurable because i assume we are working in a test environment and we'll need to update it when we are ready to go live.